README.md

```diff
@@ -1,11 +1,61 @@
+- [`let` and `const`](http://wesbos.com/let-vs-const/) - We'll convert `var` statements to these block-scoped declaration statements.
+- [Types](https://www.typescriptlang.org/docs/handbook/basic-types.html) - We'll utilize strict TypeScript typings, adding type annotations to our code where the type cannot be inferred.
+- [Advanced Types](https://www.typescriptlang.org/docs/handbook/advanced-types.html) - We'll see how union types can help us better describe our code.
+- [Enums](https://www.typescriptlang.org/docs/handbook/enums.html) - Utilizing this TypeScript data structure, we'll define a set of named constants for our news sources.
+- [Interfaces](https://www.typescriptlang.org/docs/handbook/interfaces.html) - We'll define an interface to describe the data coming back from NewsAPI in a typesafe way.
+- [Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) - JavaScript's class syntax makes working with classes much simpler. This will clean up our code a lot.
+- [TypeScript Classes](https://www.typescriptlang.org/docs/handbook/classes.html) - We'll go beyond ES classes and utilize wonderful TypeScript class features including  modifiers and non-method properties.
+- [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) - We will utilize template literals to remove ugly concatenation and utilize it for generating multi-line templates.
+- [Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) - We'll simplify inline functions with this smaller syntax.
+- [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises) - Using Promises will allow us to encapsulate asynchronous actions and work with them in a much simpler way.
+- [Asynchronous Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) - This syntactic sugar further simplifies asynchronous code by making it look synchronous.
+- [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) - We'll ditch the XMLHttpRequest (XHR) object and really simplify our request code using the superior `fetch` method.
+- [Definite Assignment Assertions](https://www.stevefenton.co.uk/2018/01/typescript-definite-assignment-assertions/) - We'll see how the TypeScript complier helps us to avoid unused/unnecessary code by alerting us to unused and unassigned class properties.
+- [Decorators](https://www.typescriptlang.org/docs/handbook/decorators.html) - We'll introduce a `deprecated` decorator when we add a new method to the service that utilized Promises over XHR and callbacks.
+
 ## Licensing Information
 
-© 2017 [JS Foundation](https://js.foundation/) & contributors. [New BSD](https://opensource.org/licenses/BSD-3-Clause) license.
+© 2018 [JS Foundation](https://js.foundation/) & contributors. [New BSD](https://opensource.org/licenses/BSD-3-Clause) license.
```

Comment:
Remove "© 2018 [JS Foundation](https://js.foundation/) & contributors." but keep the BSD license.